### PR TITLE
chore: updating ARM64 2404 image to LTS

### PR DESCRIPTION
**Pull request by artunduman**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
update arm64 2404 from daily to LTS version

Build pipeline:
https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=100547868&view=results

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:

```
none
```

### CODE REVIEW

File: .pipelines/.vsts-vhd-builder-release.yaml [hunk 1]
```diff
@@ -917,14 +917,13 @@ stages:
...
         steps:
-          # TODO(artunduman): Remove 'daily' when available
           - bash: |
               echo '##vso[task.setvariable variable=DRY_RUN]${{parameters.dryrun}}'
               echo '##vso[task.setvariable variable=OS_SKU]Ubuntu'
               echo '##vso[task.setvariable variable=OS_VERSION]24.04'
               echo '##vso[task.setvariable variable=IMG_PUBLISHER]Canonical'
-              echo '##vso[task.setvariable variable=IMG_OFFER]0001-com-ubuntu-server-noble-daily'
-              echo '##vso[task.setvariable variable=IMG_SKU]24_04-daily-lts-arm64'
+              echo '##vso[task.setvariable variable=IMG_OFFER]ubuntu-24_04-lts'
```
Comment (chengliangli0918) [inline .pipelines/.vsts-vhd-builder-release.yaml:925]: make sure the ARM64 24.04 build pipeline still works.

**Review by chengliangli0918 — APPROVED** [review]

File: .pipelines/.vsts-vhd-builder-release.yaml (hunk 1 above)
Comment (artunduman) [reply .pipelines/.vsts-vhd-builder-release.yaml:925]: it does, I attached the build link to description

Comment (artunduman) [reply .pipelines/.vsts-vhd-builder-release.yaml:925]: also thanks to Alison resolving the previous scanning issue in her PR
https://github.com/Azure/AgentBaker/pull/4803

**Review by cameronmeissner — APPROVED** [review]
@anujmaheshwari1 to approve as well

**Review by anujmaheshwari1 — APPROVED** [review]

**Pull request merged**